train.py

```diff
@@ -68,3 +75,17 @@ def train(
                 model=model, epoch=epoch, loss=loss.item(), save_dir=checkpoints_dir
             )
             print(f"model saved to {str(checkpoints_dir)}")
+
+
+if __name__ == "__main__":
+    model = UNet()
+    # TODO: Put right init params in dataset
```

Comment:
params we are currently using 

PATCH_SIZE = 16
LARGE_PATCH_SIZE = 96

TRAIN_IMAGE_INITIAL_SIZE = 400